core/src/main/resources/reference.conf

```diff
@@ -122,14 +122,25 @@ datastax-java-driver {
 
   # The policy that decides the "query plan" for each query; that is, which nodes to try as
   # coordinators, and in which order.
+  # It can be overridden in a profile. Note that the driver creates as few instances as possible:
+  # if a named profile inherits from the default profile, or if two sibling profiles have the
+  # exact same configuration, they will share a single policy instance at runtime.
+  #
+  # If there are multiple load balancing policies in a single driver instance, they work together in
+  # the following way:
+  # - each request gets a query plan from its profile's policy (or the default policy if the request
+  #   has no profile, or the profile does not override the policy).
+  # - when the policies assign distances to nodes, the driver uses the smallest assigned distance
```

Comment:
Maybe `closest` is a better way of phrasing this instead of `smallest`?   It's more clear to me that LOCAL is "closer" instead of "smaller" than REMOTE for example.